ui/app/components/clients/charts/line.ts

```diff
@@ -90,11 +90,11 @@ export default class LineChart extends Component<Args> {
     // if max is <=4, hardcode 4 which is the y-axis tickCount so y-axes are not decimals
     return [0, max <= 4 ? 4 : max];
   }
-  get timeDomain() {
```

Comment:
chart before change, the x values should end at `2/24` based on the query (in the filter toolbar) but go until `4/24`
<img width="1061" alt="Screenshot 2024-02-27 at 12 05 29 PM" src="https://github.com/hashicorp/vault/assets/68122737/fdf2745d-6dff-4f39-9f0f-22e548e0a6fe">